### PR TITLE
Created works and about myself sections

### PR DESCRIPTION
**Pull request by shinyenyi**
-Implemented all html code and styling for works and about myself section called for in the design template
-Implemented styling for button interactions

### CODE REVIEW

**Review by franclobo — COMMENTED** [review]
Hi @shinyenyi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
- Linters Errors ❌ 

You have submitted a project, that has linter errors, please correct them and submit it again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -13,27 +13,171 @@
...
+            </div>
+            <div class="language">
+              <p>html</p>
+            </div>
+            <div class="language">
+              <p>bootstrap</p>
+            </div>
+            <div class="language">
+              <p>Ruby</p>
+            </div>
```
Comment (ElsonOtake) [inline index.html]: - [ ] Please use `<ul>` and `<li>` to list of languages. It's a project requirement that should be a list of items.

File: index.html [hunk 2]
```diff
@@ -13,27 +13,171 @@
...
+      <li class="works-project">
+        <h1 class="works-project-title">Profesional Art Printing Data</h1>
+        <p class="works-project-content">
+          A daily selection of privately personalized reads; no accounts or sign-ups
+          required. has been the industry's standard
+        </p>
+        <div class="works-project-languages">
+          <div class="language">html</div>
+          <div class="language">bootstrap</div>
+          <div class="language">Ruby</div>
```
Comment (ElsonOtake) [inline index.html]: - [ ] Please use `<ul>` and `<li>` to list of languages. It's a project requirement that should be a list of items.

File: index.html [hunk 3]
```diff
@@ -13,27 +13,171 @@
...
```
Comment (ElsonOtake) [inline index.html]: - [ ] Please use `<ul>` and `<li>` to list of languages. It's a project requirement that should be a list of items.

File: index.html [hunk 4]
```diff
@@ -13,27 +13,171 @@
...
```
Comment (ElsonOtake) [inline index.html]: - [ ] Please use `<ul>` and `<li>` to list of languages. It's a project requirement that should be a list of items.

File: index.html [hunk 5]
```diff
@@ -13,27 +13,171 @@
...
```
Comment (ElsonOtake) [inline index.html]: - [ ] Please use `<ul>` and `<li>` to list of languages. It's a project requirement that should be a list of items.

**Review by ElsonOtake — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 

Hi @shinyenyi,

Good job so far! 🎉 🎉 
Nice work!! 🏆 🏆 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥲 🥲 

woohoo-youre-almost

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @shinyenyi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**